src/helpers/getCourseInfo.js

```diff
@@ -0,0 +1,26 @@
+import { mockedAuthorsList } from '../constants';
+
+export function timeConvert(n) {
+	let num = n;
```

Comment:
This function can be simplified. At least - you can remove variables, which is used only once (like num) + replace all let to const, as we do not change them. In case you will not know how to simplified it - please, let me know

Reply:
fixed, thanks